### PR TITLE
Update limits

### PR DESCRIPTION
**Pull request by briwylde08**

### CODE REVIEW

**Comment by anupsdf** [comment]
Core PR for the new limits, https://github.com/stellar/stellar-core/pull/4491

File: docs/networks/resource-limits-fees.mdx [hunk 1]
```diff
@@ -21,18 +21,21 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
...
-| Memory limit per Txn | 40 MB |
+| Soroban txn per ledger | 100 |
+| Max CPU instructions per txn | 100 million |
+| Memory limit per txn | 40 MB |
 | Ledger key size (contract storage key) | 250 bytes |
-| Ledger entry size (including Wasm entries) per Txn | 64 KB |
-| Read/Write Ledger entries per Txn | 40 read; 25 write |
-| Read/Write bytes per Txn | 200 KB read; 65 KB write |
-| Transaction size | 70 KB |
+| Ledger entry size (including Wasm entries) per txn | 129 KB |
```
Comment (anupsdf) [inline docs/networks/resource-limits-fees.mdx:28]: KiB

File: docs/networks/resource-limits-fees.mdx [hunk 2]
```diff
@@ -21,18 +21,21 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
...
+| Read/Write ledger entries per txn | 40 read; 25 write |
+| Read/Write bytes per txn | 200 KB read; 129 KB write |
```
Comment (anupsdf) [inline docs/networks/resource-limits-fees.mdx:30]: KiB

File: docs/networks/resource-limits-fees.mdx [hunk 3]
```diff
@@ -21,18 +21,21 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
...
+| Transaction size | 129 KB |
```
Comment (anupsdf) [inline docs/networks/resource-limits-fees.mdx:31]: KiB

File: docs/networks/resource-limits-fees.mdx [hunk 4]
```diff
@@ -21,18 +21,21 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
...
 | Persistent entry minimal/initial lifetime | 120 days |
 | Temporary entry minimal/initial lifetime | 1 day |
 | Max ledger entry expiration bump | 6 months |
 | Events+return value size bytes | 8 KB |
+| Max write bytes per ledger | 140 KB |
```
Comment (anupsdf) [inline docs/networks/resource-limits-fees.mdx:36]: KiB

File: docs/networks/resource-limits-fees.mdx [hunk 5]
```diff
@@ -21,18 +21,21 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
...
+| Max txs size in bytes per ledger | 130 KB |
```
Comment (anupsdf) [inline docs/networks/resource-limits-fees.mdx:37]: KiB

**Review by anupsdf — COMMENTED** [review]
Marked some settings as KiB. The state archival one stays at KB. In future, how about splitting them into transaction level limits and ledger wide limits?

**Comment by briwylde08** [comment]
@anupsdf I think that's probably a good idea. We can split them up in a separate PR.

**Review by anupsdf — APPROVED** [review]

**Comment by anupsdf** [comment]
We probably should merge this PR on October 9th when the new limits go live.

**Pull request merged**